requirements.txt

```diff
@@ -21,3 +21,4 @@ PyYAML==3.11
 pint>=0.7
 numpy
 functools32
+elasticsearch>=2.0.0,<3.0.0
```

Comment:
Please don't add `elasticsearch` to the requirements.txt file, because not all users need to install the module. Instead, add a note to `README.txt` that the `elasticsearch` module is required to use the elasticsearch publisher. Also, when you move the `elasticsearch` import inside of the elasticsearch publisher, make sure there's an understandable error message when the module isn't found.